### DISCUSSION TITLE
Displaying used disk space on a client

### DISCUSSION THREAD

**Discussion by mykgspace** [seaweedfs/seaweedfs#4360] [General]
Hello!

I have RF2 replication set in my system. After I gave the volume to the client, he moves a 500MB file to the disk. However, it shows that it actually took 1GB.

Can you please tell me whether it is possible to make the client display the occupied space on his disk, and not the actual space on the server?

Thank you in advance!

**Comment by chrislusf** [comment]
"it shows that it actually took 1GB"

where is "it"?

In the end, there are two ways to display occupied space. We have to choose one. 

Calculate the logical size involves more work. It can not be a simple "divide by 2" operation with RF2 since some replica may not exist and are read only.

**Comment by mykgspace** [comment]
Thanks for your reply!)

It- is on the client)
Before downloading the file, the client enters the command `df -h` and sees that 2GB is available to him. The client downloads a 500MB file to disk. After downloading, he re-enters the `df -h` command and sees that he did not use 500MB, but 1GB.

Thus, the client is always forced to keep in mind that, in fact, he is allocated 2 times less memory (with RF2)
